content/programming-cookbook/visual-studio.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Visual Studio
+showMetadata: true
+editable: true
+showToc: true
+order: 8
```

Comment:
You can change order to 1 because we don't need to concern order of this document.
What we need only set order 0 for `What is programming cookbook?` because we want it to be the first item under programming cookbook.

Please check the latest source code in GitHub and you will find I have update all md directly under programming cookbook has been set order to 1 except what is programming cookbook which is set to 0.

![image](https://user-images.githubusercontent.com/2938310/116436331-81124e80-a876-11eb-88bd-0fe658d4817e.png)